CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+### 1.1.0 (2021-02-24)
+
+*  Remove unused error from CreatePayoutResponse [https://github.com/paypal/Payouts-DotNet-SDK/pull/7/commits/9abd12cfbe35097e600500952f0a445295e3ea80]
```

Comment:
add a hyperlink to commit short id
refer commit directly, avoid pull request reference to commit

Reply:
done